lib/modules/manager/nuget/__fixtures__/for-bumpVersion/minimum.csproj

```diff
@@ -0,0 +1,5 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
can we inline those small fixtures? IO is expensive on windows CI tests.

Reply:
Do you mean use one fixture instead of 3?

Reply:
no, move to string constants inside the spec file, so we don't need file IO.